src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -13,7 +13,12 @@ def call
     private
 
     def create_workflows
-      parsed_workflows_yaml = YAML.safe_load(File.read(@yaml_file))
+      begin
+        parsed_workflows_yaml = YAML.safe_load(File.read(@yaml_file))
+      rescue Psych::SyntaxError
```

Comment:
Could it be useful to put the message from `Psych::SyntaxError`? I'm wondering because I don't know what exactly the message could be.

So the code would then be:
```ruby
      rescue Psych::SyntaxError => e
        raise WorkflowsYamlNotParsable, "Unable to parse .obs/workflows.yml: #{e.message}"
      end
```

Reply:
the message is what we see in the errbit error.. maybe useful for the user, not sure. Without having the yaml file is definitely not useful.

Reply:
They know that it's the `.obs/workflows.yml` file from their SCM repository, so it can help them.